debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/rest/DebeziumOracleConnectorResourceIT.java

```diff
@@ -178,4 +179,31 @@ public void testFiltersWithInvalidTableIncludeList() {
                         "The 'table.include.list' value is invalid: A comma-separated list of valid regular expressions is expected, but Dangling meta character '+' near index 0\n+\n^"));
     }
 
+    @Test
+    public void testMetricsEndpoint() {
```

Comment:
It seems to me that the test is the same for all connectors. IMHO it is about time to intorduce a common ancestor to all connector resource ITs and you could move it there.

Reply:
Well, I'd disagree in the claim of diveregence. There will be a large set of metrics that is shared by all connectors. So it makes sense to have a single method testing all common metrics and then per-connector methods to test the distinct metrics.